.vscode/settings.json

```diff
@@ -6,7 +6,7 @@
   },
   "git.ignoreLimitWarning": true,
 
-  "editor.formatOnSave": false,
+  "editor.formatOnSave": true,
```

Comment:
I was getting sick of forgetting to format my docs with black.  I know there are a few files that aren't formatted but I figure the vast majority are and you can always save without formatting if needed.

Reply:
But make sure you are pinned to the right version of black.  otherwise you will get problems. but im fine if other people dont mind.